identity-service/src/routes/notifications.js

```diff
@@ -382,10 +382,38 @@ module.exports = function (app) {
         announcementsAfterFilter
       )
 
+      let playlistUpdates = []
+      const user = await models.User.findOne({
+        attributes: ['walletAddress'],
+        where: { id: userId },
+        raw: true
+      })
+      const walletAddress = user && user.walletAddress
+      if (walletAddress) {
+        let result = await models.UserEvents.findOne({
+          attributes: ['playlistUpdates'],
+          where: { walletAddress },
+          raw: true
+        })
+        result = result && result.playlistUpdates
+        if (result) {
+          const thirtyDaysAgo = moment().utc().subtract(30, 'days').valueOf()
+          playlistUpdates = Object.keys(result)
+            .filter(playlistId =>
```

Comment:
maybe we should do some simple validation on userLastViewed here. Not that that value could necessarily be corrupt, if it was corrupt, users would never be able to dismiss the dot

Reply:
curious what kind of validation you're thinking about